content/en/docs/user-documentation/how-tos/acm-with-microshift.md

```diff
@@ -12,14 +12,14 @@ modified: "2021-11-30T12:40:37.755+01:00"
 ---
 
 ## MicroShift with Advanced Cluster Management
-Red Hat Advanced Cluster Management(RHACM) can be used to manage the lifecycle of applications running on MicroShift. Currently, this functionality is only available for AMD64 based systems. In future releases of RHACM, management functionality of applications running on MicroShift will be available on ARM based architectures.
+Red Hat Advanced Cluster Management(RHACM) can be used to manage the lifecycle of applications running on MicroShift. Currently, this functionality is only available for AMD64 based systems. In future releases of RHACM, the management functionality of applications running on MicroShift will be available on ARM based architectures.
 
-The steps below assume that RHACM has been installed on a cluster and that MicroShift is installed on a separate cluster.
+The steps below assume that RHACM has been installed on a cluster recognized as the hub cluster and that MicroShift is installed on a separate cluster recognized as the managed cluster.
 
-### Defining the cluster
-The following steps can be performed in the RHACM UI or on the CLI by following steps defined below. On the RHACM cluster, run the following command to import the MicroShift cluster:
+### Defining the managed cluster in hub cluster
+The following steps can be performed in the RHACM UI or on the CLI. On the RHACM hub cluster, run the following commands to import the MicroShift cluster as the managed cluster:
```

Comment:
"import" is already used in the following section for the process when the klusterlet connects to the management hub. I think you'll want to use "define" here like in the heading. Then again, IIUC, the RHACM documentation refers to both steps together as "import".